dm/dm/config/task.go

```diff
@@ -357,6 +368,7 @@ func NewTaskConfig() *TaskConfig {
 		Loaders:                 make(map[string]*LoaderConfig),
 		Syncers:                 make(map[string]*SyncerConfig),
 		CleanDumpFile:           true,
+		CollationCompatible:     defaultCollationCompatible,
```

Comment:
taskConfig.adjust should check if given value is legal

Reply:
ok